packages/components-react/projects/react-ssr-wrapper/tests/unit/specs/react-ssr-wrapper.spec.tsx

```diff
@@ -151,4 +151,21 @@ describe('manual test cases', () => {
 
     expect(hasTestCaseForEveryTagNameWithInternalProp).toBeTruthy();
   });
+
+  it('should render PContentWrapper correctly with conditional child component when condition is undefined', () => {
+    const Component: React.FC<{ hasAdditionalChild?: boolean | undefined }> = ({ hasAdditionalChild }) => (
+      <fromComponents.PContentWrapper>
+        <p>Ensure conditional children do not cause Errors when condition is undefined</p>
+        {hasAdditionalChild && <p>Some Text</p>}
+      </fromComponents.PContentWrapper>
+    );
+
+    const { container } = render(
+      <PorscheDesignSystemProvider>
+        <Component />
+      </PorscheDesignSystemProvider>
+    );
+
+    expect(container.firstElementChild).toBeDefined();
```

Comment:
Kind of misleading test. When you read it, this it is obviously no surprise that this element is defined.